data/json/construction.json

```diff
@@ -882,6 +882,19 @@
     "components": [ [ [ "pebble", 100 ] ] ],
     "post_terrain": "t_railroad_rubble"
   },
+  {
+    "type": "construction",
+    "description": "Build grate bridge over air",
+    "//": "Build metal grate on air.",
```

Comment:
```suggestion
```
This is just a comment entry and not necessary. If it does not add any useful information, skip it.